SellerMarket/test_docker_config.py

```
@@ -0,0 +1,223 @@
+        mock_response = MagicMock()
+        mock_response.text = '"abc123"'
+        mock_response.raise_for_status = MagicMock()
+        mock_post.return_value = mock_response
+        
+        # Import and test
+        with patch.dict(os.environ, {'OCR_SERVICE_URL': 'http://test-ocr:8080'}):
+            if 'captcha_utils' in sys.modules:
+                del sys.modules['captcha_utils']
+            from captcha_utils import decode_captcha
+            
+            result = decode_captcha("base64encodedimage")
```

Ruff (0.14.6)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)